README.adoc

```diff
@@ -1014,6 +1014,20 @@ end
 While enforcing table defaults only in Rails is suggested by many Rails developers, it's an extremely brittle approach that leaves your data vulnerable to many application bugs.
 And you'll have to consider the fact that most non-trivial apps share a database with other applications, so imposing data integrity from the Rails app is impossible.
 
+=== 3-state Boolean [[three-state-boolean]]
+
+When boolean columns are not given default value, they end up having three possible values: `true`, `false` and `NULL`.
+Boolean operators work in unexpected ways with `NULL`. Avoid such situations by always adding default values to booleans columns.
```

Comment:
Without examples it's hard to tell what you mean by this.

Reply:
Added example